backend/src/mocks/chainArchive.ts

```diff
@@ -0,0 +1,25 @@
+import ChainArchive, { ArchivedBlock } from '../chainArchive';
+
+import * as signedBlockMock from '../mocks/signedBlock.json';
+import { blockToBinary } from '../utils';
+
+export const blocksMock = [
+  blockToBinary(signedBlockMock),
+  blockToBinary(signedBlockMock),
+  blockToBinary(signedBlockMock),
+]
+
+class ChainArchiveMock {
```

Comment:
Ideally this should have `class X implements Y` and `Y` should be a part of the API that is being relied on instead of doing type casing below.